### ISSUE TITLE
Weird space above tabs

### ISSUE THREAD

**Issue by ghost** [Centril/firefox-movable-titlebar#1]
above the tabs there is about half a cm of space.. Would you be able to fix it?

**Comment by Centril** [comment]
Thanks for reporting!

Could you post a screenshot of this? Also: what other addons are you using? [firefox-line](/Centril/firefox-line) perhaps?

Cheers!

**Comment by ghost** [comment]
http://i.imgur.com/qZJEVdo.png

It happens even if I disable all addons as well.

**Comment by Centril** [comment]
OK, so your setup is: normal firefox (not dev edition), windows 10 and no addons... Correct?

This is quite different than my own, so I'll have to do some digging and/or create a more intelligent way to handle that part that uses javascript instead of CSS.

**Comment by ghost** [comment]
Yep, normal firefox, windows 10, happens with addons and without.

**Comment by Centril** [comment]
Right, I only tested it on firefox-dev-edition with & without [firefox-line](/Centril/firefox-line).

**Comment by tdim** [comment]
Space above toolbar is also present in FF nightly 49.0a1, but only in maximized mode
(on windows 10, no plugins enabled, no additional extensions) 

Picture attached
![2016-04-26_215702](https://cloud.githubusercontent.com/assets/16364621/14839594/1ac298b0-0bfa-11e6-8d3d-d9eaee8bd78a.png)

**Comment by Centril** [comment]
The problem is that firefox-dev-edition and normal firefox don't behave the same way, so I guess I have to add a check that does it differently for normal firefox.

Unfortunately I don't have time to fix this issue atm, but if you can make a pull request that'd be nice.
